axelrod/tests/strategies/test_titfortat.py

```diff
@@ -563,3 +563,54 @@ def test_strategy(self):
         actions = [(C, C), (C, C), (C, D), (C, D), (D, C), (D, D), (D, D),
                     (D, C), (D, C), (C, D), (C, D)]
         self.versus_test(opponent, expected_actions=actions)
+
+class TestAlexei(TestPlayer):
+    """
+    Note that this test is referred to in the documentation as an example on
+    writing tests.  If you modify the tests here please also modify the
+    documentation.
+    """
+
+    name = "Alexei"
```

Comment:
Now that your are using `@FinalTransformer` the name of your strategy is different.
`AssertionError: 'Alexei: D' != 'Alexei'`

Take a look at the backstabber tests https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/tests/strategies/test_backstabber.py.